### ISSUE TITLE
[DOCS]ERROR: This CPU does not provide an interface to manipulate the Uncore frequency.

### ISSUE THREAD

**Issue by shawnonthemove** [RRZE-HPC/likwid#446]
I  have a problem about the default settings of **likwid-setFrequencies -umax xx**.
The thing is i am doing some jobs about UFS(uncore frequency scaling), i use **likwid-perctr** to collect datasets and adjust socket 0' uncore frequency every 1.5s.
After about ten times of adjustments, the error came out, 
_**ERROR: This CPU does not provide an interface to manipulate the Uncore frequency.**_
and i looked over the frequencies through **likwid-setFrequencies -p**
the uncore frequencies are 0.0. After I reboot Ubuntu, it is all set. 
![截屏2021-12-23 上午11 42 32](https://user-images.githubusercontent.com/86863023/147184756-c85ce2de-9bcb-4ca3-9e6c-44654fb1bfea.png)
So, i am wondering whether there are some protection policies about it or not, and how can i turn it off.
I really need to do the adjust actions every 1 or 1.5s, 
I'm always looking forward to your reply at your earliest convenience.

**Comment by TomTheBear** [comment]
Thanks for the report, I'll check that after the Christmas days. There is not rate limiting or any other protection active. I assume there is some problem when setting the max. frequency.

Please provide the output of `likwid-setFrequencies -V 3 -umax xx`

:christmas_tree:

**Comment by TomTheBear** [comment]
I cannot reproduce it on my local system. I increased the maximal Uncore frequency every second:

```
$ echo "$FREQS"
0.4 0.5 0.6 0.7 0.8 0.9 1.0 1.1 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7 2.8 2.9 3.0 3.1 3.2 3.3 3.4
$ for F in $FREQS; do
  likwid-setFrequencies -umax "$F" 2>&1 | grep -v "WARN";
  likwid-setFrequencies -p 2>&1 | grep "min/max";
  sleep 1;
done
Socket 0: min/max 0.4/0.4 GHz
Socket 0: min/max 0.4/0.5 GHz
Socket 0: min/max 0.4/0.6 GHz
Socket 0: min/max 0.4/0.7 GHz
Socket 0: min/max 0.4/0.8 GHz
Socket 0: min/max 0.4/0.9 GHz
Socket 0: min/max 0.4/1.0 GHz
Socket 0: min/max 0.4/1.1 GHz
Socket 0: min/max 0.4/1.2 GHz
Socket 0: min/max 0.4/1.3 GHz
Socket 0: min/max 0.4/1.4 GHz
Socket 0: min/max 0.4/1.5 GHz
Socket 0: min/max 0.4/1.6 GHz
Socket 0: min/max 0.4/1.7 GHz
Socket 0: min/max 0.4/1.8 GHz
Socket 0: min/max 0.4/1.9 GHz
Socket 0: min/max 0.4/2.0 GHz
Socket 0: min/max 0.4/2.1 GHz
Socket 0: min/max 0.4/2.2 GHz
Socket 0: min/max 0.4/2.3 GHz
Socket 0: min/max 0.4/2.4 GHz
Socket 0: min/max 0.4/2.5 GHz
Socket 0: min/max 0.4/2.6 GHz
Socket 0: min/max 0.4/2.7 GHz
Socket 0: min/max 0.4/2.8 GHz
Socket 0: min/max 0.4/2.9 GHz
Socket 0: min/max 0.4/3.0 GHz
Socket 0: min/max 0.4/3.1 GHz
Socket 0: min/max 0.4/3.2 GHz
Socket 0: min/max 0.4/3.3 GHz
Socket 0: min/max 0.4/3.4 GHz
```

I will retry it with a two socket system, maybe the issue occurs there.

**Comment by shawnonthemove** [comment]
I tried to test the problem just like you did, and it works.
So I tried to trace the problem on my codes,
I use python to code, and passed a string type variable to my shell script through `%s % freq`, 
it looks like,
![图片1](https://user-images.githubusercontent.com/86863023/147554096-22504c8a-f72e-4813-9f33-c5219c60f32d.png)
the freq param became a string with many decimal places, and then I replaced the code with `%.1f % freq`,
It is like  freq: 1.5, and the problem is solved!
So the problem is caused by the freq param, the command `likwid-setFrequencies -umax xx` refused to take over a param with many decimal places.
All in all, thanks very much for your reply sincerely!
